suite-native/connection-status/src/OfflineHeader.tsx

```diff
@@ -0,0 +1,45 @@
+
+import { useSafeAreaInsets } from 'react-native-safe-area-context';
+import Animated, { SlideInUp, SlideOutUp } from 'react-native-reanimated';
+
+import { Icon } from '@suite-common/icons';
+import { Text, HStack } from '@suite-native/atoms';
+import { Translation } from '@suite-native/intl';
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+
+import { useIsOfflineHeaderVisible } from './useIsOfflineHeaderVisible';
+
+const containerStyle = prepareNativeStyle(utils => ({
+    backgroundColor: utils.colors.backgroundAlertYellowBold,
+    alignItems: 'center',
+}));
+
+const contentStyle = prepareNativeStyle<{ topSafeAreaInset: number }>(
+    (_, { topSafeAreaInset }) => ({
+        marginTop: topSafeAreaInset,
+        height: 40,
```

Comment:
Hardcoded heigh is not responsive 🙉 What about changed font size?

Reply:
6838ae7198